src/modules/app/reducers/index.js

```diff
@@ -31,8 +31,17 @@ const endpoints = [
   poisEndpoint
 ]
 
-export const startFetchReducer = <T> (oldPayload: Payload<T>, action: ActionType<T>): Payload<T> => action.payload
-export const finishFetchReducer = <T> (oldPayload: Payload<T>, action: ActionType<T>): Payload<T> => {
+// fixme: WEBAPP-400 This type should be removed
+export type ReducerType<T> = <T> (oldPayload?: Payload<T>, action: ActionType<T>) => Payload<T>
```

Comment:
I think `oldPayload` should always be supplied, even if it is `null` or `undefined`, so `oldPayload: ?Payload<T>` would be better imo.